### PR TITLE
Feature/create invoice overview

### PR DESCRIPTION
**Pull request by Jobbiesaus**
# Description
Added an overview of the open invoice accounts at the top of the invoice tab.

## Related issues/external references
Fixes#461
Fixes#130

## Types of changes
- New feature _(non-breaking change which adds functionality)_

### CODE REVIEW

File: apps/dashboard/src/components/FindUser.vue [hunk 1]
```diff
@@ -77,17 +106,30 @@ const filterUsers = (e: any) => {
...
-  emits('update:value', selectedUser.value);
+  emits('update:user', selectedUser.value);
 });
+
 </script>
 
 <style scoped lang="scss">
+.broke {
 
+}
```
Comment (JustSamuel) [inline apps/dashboard/src/components/FindUser.vue]: ?

File: apps/dashboard/src/mixins/openInvoiceAccountsMixin.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import type { BaseUserResponse } from "@sudosos/sudosos-client";
+
+
+
+export function useOpenInvoiceAccounts() {
+    const userStore = useUserStore();
+    const openInvoiceAccounts = computed(() => {
+            return Object.values(userStore.users).filter(isNotZeroAndInvoice).length;
+        }
+    );
```
Comment (JustSamuel) [inline apps/dashboard/src/mixins/openInvoiceAccountsMixin.ts:15]: is this fast enough? maybe we should add a dedicated endpoint for this in the backend?

**Review by JustSamuel — CHANGES_REQUESTED** [review]
Solid work! love that you added the notifications ❤️

File: apps/dashboard/src/components/FindUser.vue (hunk 1 above)
Comment (Jobbiesaus) [reply apps/dashboard/src/components/FindUser.vue]: This was my IDE who did something weird for me. Should be gone now.

**Review by JustSamuel — APPROVED** [review]

File: apps/dashboard/src/mixins/openInvoiceAccountsMixin.ts (hunk 2 above)
Comment (Jobbiesaus) [reply apps/dashboard/src/mixins/openInvoiceAccountsMixin.ts:15]: Let's try it out

**Pull request merged**